src/api/admin.ts

```diff
@@ -12,7 +12,7 @@ import {
   VenueTemplate,
 } from "types/venues";
 
-import { WithId } from "utils/id";
+import { WithId, WithWorldId } from "utils/id";
```

Comment:
@mike-lvov @denisdimitrov Does it make sense for these utility mixins to be generics or is it easier to deal with them as type unions?